model/host/host.go

```diff
@@ -99,6 +99,9 @@ type Host struct {
 	HasContainers bool `bson:"has_containers,omitempty" json:"has_containers,omitempty"`
 	// stores the ID of the host a container is on
 	ParentID string `bson:"parent_id,omitempty" json:"parent_id,omitempty"`
+
+	// SpawnOptions holds data which the monitor uses to determine when to terminate hosts spawned by tasks.
+	SpawnOptions SpawnOptions `bson:"spawn,omitempty" json:"spawn,omitempty"`
```

Comment:
Should this be a pointer? Also maybe "spawn_options" instead?

Reply:
Made it "spawn_options". I don't see a reason for it to be a pointer though.